src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int num;
```

Comment:
```suggestion
    private final int number;
```
Its better to use full names for variables